docs/CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Implement tracker SLA start date setting (OSIDB-1393)
 - Implement tracker SLA end date setting (OSIDB-96)
 - Properly link Jira trackers to flaws on creation and update (OSIDB-1426)
+- Add OSV collector (OSIDB-677)
```

Comment:
We usually update README for user-visible changes only. I am not sure if this is that case.

Reply:
Good point. I would probably leave this there however. It a significant change and should be eventually user-facing when they start processing the collected data collected from OSV.